token/cli/src/main.rs

```diff
@@ -2160,6 +2165,14 @@ fn app<'a, 'b>(
                             Rate authority defaults to the mint authority."
                         ),
                 )
+                .arg(
+                    Arg::with_name("enable_non_transferable")
+                        .long("enable-non-transferable")
+                        .takes_value(false)
```

Comment:
```suggestion
                        .alias("enable-nontransferable")
```

i have ptsd from cargo doing `--no-run` but `--nocapture`